google/cloud/resource_manager/project.py

```diff
@@ -102,6 +102,20 @@ def path(self):
         """URL for the project (ie, ``'/projects/purple-spaceship-123'``)."""
         return "/%s" % (self.full_name)
 
+    @property
+    def ancestry(self):
+        """Returns the project ancestry information.
+
+        See
+        https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/getAncestry
+        """
+        client = self._require_client(self._client)
+        resp = client._connection.api_request(
+            method="POST",
+            path='%s:getAncestry?alt=json' % (self.path),
+        )
```

Comment:
JSON is the default response type for `getAncestry`.
```suggestion
        resp = client._connection.api_request(
            method="POST", path='%s:getAncestry' % (self.path),
        )
```